services/reporting/reporting.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+package reporting
+
+import (
+	"time"
+
+	"github.com/getsentry/sentry-go"
+)
+
+// Return the options that can be configured for the error reporting
+// library.
+func Options() sentry.ClientOptions {
+	return sentry.ClientOptions{}
+}
+
+// Initialize the reporting services.
+func Init(options sentry.ClientOptions) error {
```

Comment:
Here I would prefer to not leak the `sentry` types, I would prefer to receive the data that we need to configure our sentry and create the `sentry.ClientOptions` object from there.

Reply:
See my comment above, the leak of types is intentional but I can fix it ;)